### PR TITLE
DNSChaos e2e test: increase timeout time

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

### What problem does this PR solve?

fix issue https://github.com/chaos-mesh/chaos-mesh/issues/3113

### What's changed and how it works?

just increase timeout time

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [x] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [x] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

File: e2e-test/e2e/chaos/dnschaos/dns.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func TestcaseDNSRandom(
 	framework.ExpectNoError(err, "create dns chaos error")
 
 	for _, domainName := range effectDomainNames {
-		err = wait.Poll(time.Second, 5*time.Second, func() (done bool, err error) {
+		err = wait.Poll(time.Second, 10*time.Second, func() (done bool, err error) {
```
Comment (cwen0) [inline e2e-test/e2e/chaos/dnschaos/dns.go:83]: I remember this issue happened in recent days. Why does this issue not happen before?

**Review by Hexilee — APPROVED** [review]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**